great_expectations/validator/validator.py

```diff
@@ -395,7 +395,7 @@ def _build_expectation_configuration(
         meta: dict,
         expectation_impl: "Expectation",  # noqa: F821
     ) -> ExpectationConfiguration:
-        auto: Optional[bool] = expectation_kwargs.get("auto")
```

Comment:
When possible, let's avoid nullable values. This is only ever used in a boolean context so I've changed this.